pnpm-workspace.yaml

```
@@ -1,9 +1,9 @@
 packages:
   # all packages in subdirs of packages/ and components/
-  - 'providers/*'
+  - 'providers/**'
   - 'packages/*'
-  - 'enterprise/packages/*'
+  - 'enterprise/packages/**'
```

In line with your previous suggestion @p-fernandez , what do you think of this? I think it's better to be fully explicit about the folder depth to avoid any doubt about which folders are targeted. `**` doesn't produce explicitness.

```suggestion
  - 'enterprise/packages/**'
  + 'enterprise/packages/*'
  + 'enterprise/libs/*'
```

---

For this change we will need to do two PRs in a row.
I address some of the submodule changes here now.
Then we will go on here: https://github.com/novuhq/packages-enterprise/pull/16
And then finally trim the things back in the monorepo after syncing.
The goal is to move the `libs` package one level down as agreed.

CC: @scopsy